Components/Alert/alert.css

```diff
@@ -0,0 +1,117 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
```

Comment:
you can remove this as you added in main.css

Reply:
Hey, actually main.css is for my website only users can't import main.css they can only import component CSS files so they won't get box-sizing property in that CSS. That's why I kept it there, as user might import CSS file for only 1 component also